goosepaper/multiparser.py

```diff
@@ -27,35 +37,150 @@ def __init__(self):
             required=False,
             help="The output file path at which to save the paper",
         )
+        self.parser.add_argument(
+            "-f",
+            "--folder",
+            required=False,
+            help="Folder to which the document will be uploaded in your remarkable.",
+        )
         self.parser.add_argument(
             "-u",
             "--upload",
             action="store_true",
             required=False,
+            default=None,
             help="Whether to upload the file to your remarkable using rmapy.",
         )
+        self.parser.add_argument(
+            "--noupload",
```

Comment:
I think `--upload/--noupload` should be [mutually exclusive](https://docs.python.org/3/library/argparse.html#mutual-exclusion) to avoid the weirdness of a command like `goosepaper --upload --noupload myfile.pdf`

Reply:
That would require some significant mods to the multiparser to determine if upload is present on the command line only or in the config file. My thought process also was the allowing '--noupload' combined with --showconfig on the command line would allow easier testing/validation of configs without having to tweak config files back and forth. 

Poor logic on my part?